src/planner/match/PropIndexSeek.cpp

```diff
@@ -60,19 +61,22 @@ StatusOr<SubPlan> PropIndexSeek::transformNode(NodeContext* nodeCtx) {
     auto contexts = std::make_unique<std::vector<IQC>>();
     contexts->emplace_back(std::move(iqctx));
     auto columns = std::make_unique<std::vector<std::string>>();
+    columns->emplace_back(kVid);
     auto scan = IndexScan::make(matchClauseCtx->qctx,
                                 nullptr,
                                 matchClauseCtx->space.id,
                                 std::move(contexts),
                                 std::move(columns),
                                 false,
                                 nodeCtx->scanInfo.schemaId);
+    scan->setColNames({kVid});
```

Comment:
What is the purpose here, and why is it not the same as below?
```
returnCols_->emplace_back(kVid);
colNames_.emplace_back(kVertexID);
```

Reply:
Let user reference result of lookup.

Reply:
E.G. `lookup ... | FETCH ON PROP $-.VertexID`.

Reply:
And make the column names same as 1.0

Reply:
> Let user reference result of lookup.

Why  different with `IndexScanValidator::prepareYield`?

Reply:
The prepareYield won't affect the column names of result, It's a bug I will fix in latter PR.